README.md

```diff
@@ -690,3 +690,146 @@ In general, `for of` is recommended over `for in` or `forEach`
 </details>
 
 ---
+
+###### 19. Array Manipulation: Understanding array.push() Method in JavaScript
+
+```javascript
+let arr = [1, 2, 3, 4];
+let result = arr.push(5);
+console.log(result);
+console.log(arr);
+```
+
+What does the above code snippet output?
+
+- A: 'result' will be 5 and 'arr' will be [1, 2, 3, 4, 5]
+- B: 'result' will be 5 and 'arr' will be [5, 1, 2, 3, 4]
+- C: 'result' will be 4 and 'arr' will be [2, 3, 4, 5]
+- D: 'result' will be 4 and 'arr' will be [5, 2, 3, 4]
+
+<details><summary><b>Answer</b></summary>
+<p>
+
+#### Answer: A
+
+The `array.push()` method in JavaScript adds one or more elements to the end of an array and returns the new length of the array. In the given code, arr is an array with the values `[1, 2, 3, 4]`. The `push()` method is called with the argument 5, which adds `5` to the end of the arr array.
+
+The `push()` method returns the new length of the array after the addition of the element(s). In this case, the new length of arr will be 5 because `5` is added to the end of the array.
+
+</p>
+</details>
+
+---
+
+##### 20. Array Manipulation: Understanding array.unshift() Method in JavaScript
+
+```javascript
+let arr = [3, 5, 7, 9];
+let result = arr.unshift(1, 2);
+console.log(result);
+console.log(arr);
+```
+
+What does the above code snippet output?
+
+- A: 'result' will be 6 and 'arr' will be [3, 5, 7, 9, 1, 2]
```

Comment:
Please update options for better readability. Also I have suggested changes for option D to make it more challenging for users who might think `unshift` will return an array instead of length.

```diff
- - A: 'result' will be 6 and 'arr' will be [3, 5, 7, 9, 1, 2]
- - B: 'result' will be 6 and 'arr' will be [1, 2, 3, 5, 7, 9]
- - C: 'result' will be 4 and 'arr' will be [1, 2, 3, 5]
- - D: 'result' will be 4 and 'arr' will be [7, 9, 1, 2]

+ - A: result: 6 arr: [3, 5, 7, 9, 1, 2]
+ - B: result: 6 arr: [1, 2, 3, 5, 7, 9]
+ - C: result: 4 arr: [1, 2, 3, 5]
+ - D: result: [1, 2, 3, 5, 7, 9] arr: [1, 2, 3, 5, 7, 9]
```